src/System.Runtime.Serialization.Primitives/src/System/Runtime/Serialization/FormatterServices.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+namespace System.Runtime.Serialization
+{
+    public static class FormatterServices
+    {
+    	public static object GetUninitializedObject(Type type)
+    	{
+    		// On .NET Native, this is not implemented
+    		throw new NotImplementedException();
+    	}
+    }
+}
```

Comment:
After discussion with @ericstj , we decided it was best to just throw when in .NET Native, because of some complications we encountered doing something more elaborate. It may be that the underlying functionality is already there -- that's something to potentially look at later.